imports/plugins/core/orders/client/templates/workflow/shippingTracking.js

```diff
@@ -193,6 +193,7 @@ Template.coreOrderShippingTracking.helpers({
     const order = Template.instance().order;
     const shipment = order.shipping[0];
 
-    return shipment.packed && shipment.tracking;
+    return _.includes(shipment.workflow.workflow, "coreOrderWorkflow/packed") && shipment.tracking ||
+    _.includes(shipment.workflow.workflow, "coreOrderWorkflow/packed");
```

Comment:
this feels brittle as well. What if there is a custom workflow that doesn't include `coreOrderWorkflow/packed` (e.g. uses a different workflow, or pre-packed items, etc.)